app/src/androidTest/java/org/simple/clinic/TestData.kt

```diff
@@ -195,7 +195,7 @@ class TestData @Inject constructor(
       deletedAt: Instant? = null,
       address: PatientAddressPayload = addressPayload(),
       phoneNumbers: List<PatientPhoneNumberPayload>? = listOf(phoneNumberPayload()),
-      businessIds: List<BusinessIdPayload>? = listOf(businessIdPayload(patientUuid = uuid))
+      businessIds: List<BusinessIdPayload> = listOf(businessIdPayload(patientUuid = uuid))
```

Comment:
I've realised that when we are generating test business IDs, we don't generate metadata for it. In the test, we set the meta version to `org.simple.clinic.bppassport.meta.v1`, but set the meta to an empty string. Since this is meant to be an integration test, do you feel that we should generate metadata appropriately?

Reply:
I have seen that we are not using `metadata` anywhere.
It is a `String` and we are assigning it `""` which seems fine for now. 

If we ever we start using the `metadata`, we will change this then.

Reply:
Integration tests are not just for us, they're also for the server so that they get feedback if any contracts are broken. Our contract is basically that if we set a meta version, there also needs to be a corresponding meta data, which is why I believe we should send it. Does that make sense?

Reply:
Oh ok. I did not know that.
I guess we should do it then.